### DISCUSSION TITLE
NixtlaClient.forecast() got an unexpected keyword argument 'feature_contributions'

### DISCUSSION THREAD

**Discussion by zoeouyang2543** [Nixtla/nixtla#476] [Q&A] [answered]
when i add feature contributions as attribute ,i met the problem: NixtlaClient.forecast() got an unexpected keyword argument 'feature_contributions'.the version of nixtla is 0.6.0
please give me a hand.thank you

**Comment by zoeouyang2543** [comment]
@cchallu please help

**Comment by elephaint** [comment]
Hi, thanks for using TimeGPT.

This is strange, as of v0.6.0 the feature_contributions argument should work. How did you verify the version of nixtla? 

Could you post:
1. The exact error message you receive?
2. A piece of code that you are running that gives the error?

**Reply by zoeouyang2543** [reply, marked as answer]
> Hi, thanks for using TimeGPT.
> 
> This is strange, as of v0.6.0 the feature_contributions argument should work. How did you verify the version of nixtla?
> 
> Could you post:
> 
> 1. The exact error message you receive?
> 2. A piece of code that you are running that gives the error?

sorry,when i reboot the mechine,the problem is solved.thanks for your reply

**Reply by elephaint** [reply]
Great to hear!
